add/backend/addBackend.js

```diff
@@ -51,7 +51,7 @@ const createBackend = (mode, serverTestingSelection, databaseSelection) => {
 
   addDatabase(databaseSelection)
```

Comment:
addDatabase and testBackend in addBackend should definitely be checked to see if they need an await.